src/screens/Authentication/PhoneNumber/PhoneNumber.tsx

```diff
@@ -9,37 +9,44 @@ import {
 } from 'react-hook-form';
 import { FirebaseRecaptchaVerifierModal } from 'expo-firebase-recaptcha';
 import PhoneInput from 'react-native-phone-number-input';
+import { AuthStackScreenProps } from '../../../types/navigation';
 import styles from './styles';
 import {
   phoneGetConfirmation,
   getActivationStatus,
 } from '../../../firebase/auth';
 import { firebaseApp } from '../../../firebase/config';
 import StyledButton from '../../../components/StyledButton/StyledButton';
-import logo from '../../../assets/cnsc-logo.png';
-import WelcomeScreen from '../Welcome/Welcome';
-import { AuthStackScreenProps } from '../../../types/navigation';
+
+const logo = require('../../../assets/cnsc-logo.png');
 
 function PhoneNumberScreen({
   navigation,
 }: AuthStackScreenProps<'PhoneNumberScreen'>) {
   interface FormValues {
     phoneNumber: string;
   }
+
   const { ...methods } = useForm();
   const recaptchaVerifier = useRef(null);
   const [phoneNumber, setPhoneNumber] = useState('');
   const [valid, setValid] = useState(false);
+  const [phoneError, setPhoneError] = useState('');
   const phoneInput = useRef<PhoneInput>(null);
 
   const onSubmit: SubmitHandler<FormValues> = async () => {
     try {
-      const checkValid = phoneInput.current?.isValidNumber(phoneNumber);
+      const validatePhoneNumber = () => {
+        if (phoneInput.current?.isValidNumber(phoneNumber)) {
+          setPhoneError('');
+          return true;
+        } setPhoneError('Oops! Invalid phone number. Please try again.');
+      };
+      setValid(validatePhoneNumber());
       console.log(valid);
-      setValid(checkValid ?? false);
       // To Do: render error label
       const activated = await getActivationStatus(phoneNumber);
```

Comment:
can probably do a check to see if phone number valid first and then call the getActivationStatus b/c don't think we should call this if the phone number is already known to be invalid